library/common/jni/jni_utility.h

```diff
@@ -14,6 +14,22 @@ JavaVM* get_vm();
 
 JNIEnv* get_env();
 
+void set_class_loader(jobject class_loader);
+
+/**
+ * Finds a class with a given name using a class loader provided with the use
+ * of `set_class_loader` function. The class loader is supposed to come from
+ * application's context and should be associated with project's code - Java classes
+ * defined by the project. For finding classes of Java built in-types use
+ * `env->FindClass(...)` method instead as it is lighter to use.
+ *
+ * @param class_name, the name of the class to find (i.e. "org.chromium.net.AndroidNetworkLibrary").
```

Comment:
its / delimited, not . right?

Reply:
It requires `.` - I added an example to avoid confusion since I realize that `/'` would be a more obvious choice here.

Reply:
oh i see, you're calling loadClass and not findClass

nice consistency java